### DISCUSSION TITLE
why I can't seed anything when I installed?

### DISCUSSION THREAD

**Discussion by chrixtal** [jgraichen/redmine_dashboard#270] [Q&A]
After I installed, I can't seen on plugin and my settings, permissions. why?

I can't find any doc for how to shown on my redmine 5.0.2

**Comment by jgraichen** [comment]
Please provide some details on what you did and how you installed the plugin, logs showing commands and output, and screenshots on what you are seeing.

Otherwise, there is not much I can help you with.
